system_strings/sysmes_text.en

```diff
@@ -4261,14 +4261,14 @@ Even if I want to look around the mansion, I should greet Akiha first.
 -- 10分もあれば屋敷を見て回れる。^ロビーに足を踏み入れた矢先、^声をかけてきたのは……。
 // @flowchart_descr
 Ten minutes is plenty for a quick snoop around the mansion.
- As soon as I step out into the lobby, I'm greeted by―――
+ As soon as I step out into the lobby, I'm greeted by――
 }
 [sha:526b7c11e67f9f75342c6d2d37a2c091c3ebb82e]{
 -- 子供の頃は覗く事もできなかった^遠野家当主の私室、執務室。^そこに妹、秋葉が待っている。
 // @flowchart_descr
 The office of the Tohno family head is private.
  As a child, I wasn't even allowed to look inside.
- Now, my younger sister Akiha awaits me there.
+ Now, my younger sister, Akiha awaits me there.
```

Comment:
This comma doesn't make sense to me

Reply:
Doesn't to me too after thinking about it
![image](https://github.com/Tsukihimates/Tsukihime-Translation/assets/90473809/09ab89a1-3548-4941-8456-347020cc653a)